Assets/Scripts/WorldEngine/World.cs

```diff
@@ -1384,15 +1384,15 @@ public long Update()
 
         Profiler.EndSample();
 
-        Profiler.BeginSample("UpdateFactions");
+        Profiler.BeginSample("RemoveFactions");
 
-        UpdateFactions();
+        RemoveFactions();
 
         Profiler.EndSample();
 
-        Profiler.BeginSample("RemoveFactions");
+        Profiler.BeginSample("UpdateFactions");
 
-        RemoveFactions();
+        UpdateFactions();
```

Comment:
I'm guessing this was the fix for #119?

Reply:
Yes, turns out I never remove factions during faction updates, so the fix was simply to do faction updates after faction removals. I probably could do the same with polities.